hack/update_crdb_versions_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2021 The Cockroach Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
We typically put tests in their own package to ensure we test the public API (e.g. `package main_test`). You can look at [//hack/release] or [//hack/smoketest] for examples of making this easier.

[//hack/release]: https://github.com/cockroachdb/cockroach-operator/tree/master/hack/release
[//hack/smoketest]: https://github.com/cockroachdb/cockroach-operator/tree/master/hack/smoketest